File: Python/Disk.py
class Disk():

    #Constructor
    def __init__(self):
        
        self.tipe = "" #deklarasi atribut
        self.kapasitas = "" #deklarasi atribut
        self.harga = 0 #deklarasi atribut

    #set get
    def setTipe(self, tipe):
        self.tipe = tipe
    def getTipe(self):
        return self.tipe
    def setKapasitas(self, kapasitas):
        self.kapasitas = kapasitas
    def getKapasitas(self):
        return self.kapasitas
    def setHarga(self, harga):
        self.harga=  harga
    def getHarga(self):
        return self.harga
    
    #keluaran
    def printDisk(self):
        print("---- DISK ----")
        print("Type: " + str(self.tipe))
        print("Capacity: " + str(self.kapasitas))
        print("Price: " + str(self.harga))

File: PHP/Disk.php
<?php
class Disk{
    
    private $tipe; //deklarasi Attribute
    private $kapasitas; //deklarasi Attribute
    private $harga; //deklarasi Attribute

    #Constructor 
    function __construct($tipe, $kapasitas, $harga)
    {
        $this->tipe = $tipe;
        $this->kapasitas = $kapasitas;
        $this->harga = $harga;
    }
    #set get
    function setTipe($tipe){
        $this->tipe = $tipe;
    }
    function getTipe(){
        return $this->tipe;
    }
    function setKapasitas($kapasitas){
        $this->kapasitas = $kapasitas;
    }
    function getKapasitas(){
        return $this->kapasitas;
    }
    function setHarga($harga){
        $this->harga = $harga;
    }
    function getHarga(){
        return $this->harga;
    }
    
    #keluaran
    function printDisk(){
        echo "---- DISK ----". "<br>";
        echo "Type: " .$this->tipe."<br>";
        echo "Capacity: " .$this->kapasitas."<br>";
        echo "Price: " .$this->harga."<br>";
    }

    function __destruct()
    {
        
    }
}
?>